### PR TITLE
Updates to spring cloud Greenwich.SR3

### PR DESCRIPTION
**Pull request by fangjian0423**
As titled.

Refer https://spring.io/blog/2019/09/12/spring-cloud-greenwich-sr3-released.

### CODE REVIEW

**Comment by fangjian0423** [comment]
close #938

File: pom.xml [hunk 1]
```diff
@@ -71,11 +71,11 @@
...
-        <spring-cloud-commons.version>2.1.2.RELEASE</spring-cloud-commons.version>
-        <spring-cloud-netflix.version>2.1.2.RELEASE</spring-cloud-netflix.version>
-        <spring-cloud-openfeign.version>2.1.2.RELEASE</spring-cloud-openfeign.version>
-        <spring-cloud-bus.version>2.1.2.RELEASE</spring-cloud-bus.version>
-        <spring-cloud-gateway.version>2.1.2.RELEASE</spring-cloud-gateway.version>
+        <spring-cloud-commons.version>2.1.3.RELEASE</spring-cloud-commons.version>
+        <spring-cloud-netflix.version>2.1.3.RELEASE</spring-cloud-netflix.version>
+        <spring-cloud-openfeign.version>2.1.3.RELEASE</spring-cloud-openfeign.version>
+        <spring-cloud-bus.version>2.1.3.RELEASE</spring-cloud-bus.version>
+        <spring-cloud-gateway.version>2.1.3.RELEASE</spring-cloud-gateway.version>
```
Comment (ly641921791) [inline pom.xml]: Is it possible to add an attribute here like this?
`<spring-cloud-component.version>2.1.3.RELEASE</spring-cloud-component.version>`

Other spring cloud components introduce versions through this property, like this
`<spring-cloud-commons.version>${spring-cloud-component.version}</spring-cloud-commons.version>`

File: spring-cloud-alibaba-dubbo/pom.xml [hunk 2]
```diff
@@ -14,8 +14,8 @@
     <name>Spring Cloud Alibaba Dubbo</name>
 
     <properties>
-        <spring-cloud-zookeeper.version>2.1.2.RELEASE</spring-cloud-zookeeper.version>
-        <spring-cloud-consul.version>2.1.2.RELEASE</spring-cloud-consul.version>
+        <spring-cloud-zookeeper.version>2.1.3.RELEASE</spring-cloud-zookeeper.version>
+        <spring-cloud-consul.version>2.1.3.RELEASE</spring-cloud-consul.version>
```
Comment (ly641921791) [inline spring-cloud-alibaba-dubbo/pom.xml]: Why not put all the versions in the dependencies module to manage it?

**Comment by fangjian0423** [comment]
@ly641921791 hope you can create a pr to finish it. Thanks

**Comment by ly641921791** [comment]
> @ly641921791 hope you can create a pr to finish it. Thanks

Now that there are multiple branches, should I submit pr to the master?

**Comment by fangjian0423** [comment]
> > @ly641921791 hope you can create a pr to finish it. Thanks
> 
> Now that there are multiple branches, should I submit pr to the master?

Do it in master branch.

**Pull request merged**